django/db/backends/oracle/schema.py

```diff
@@ -124,6 +126,19 @@ def _alter_field_type_workaround(self, model, old_field, new_field):
         self.remove_field(model, old_field)
         # Rename and possibly make the new field NOT NULL
         super().alter_field(model, new_temp_field, new_field)
+        # Recreate foreign key (if necessary) because the old field is not
+        # passed to the alter_field() and data types of new_temp_field and
+        # new_field always match.
+        new_type = new_field.db_type(self.connection)
+        if (
+            self.connection.features.supports_foreign_keys and (
```

Comment:
Is there a purpose in consulting this flag that's always True on Oracle?

Reply:
Eagle eye :eagle: Thanks, removed.